components/pkg-export-kubernetes/plan.sh

```diff
@@ -6,9 +6,9 @@ pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_license=('Apache-2.0')
 pkg_deps=(core/docker)
 pkg_build_deps=(
-  core/musl core/zlib-musl core/xz-musl core/bzip2-musl core/libarchive-musl
-  core/openssl-musl core/libsodium-musl
-  core/coreutils core/rust core/gcc core/make
+  core/zlib core/xz core/bzip2 core/libarchive
```

Comment:
We'll need to move some of these into `pkg_deps` so they install at runtime. The list will end up looking pretty similar to `hab-sup`'s dep list.